lib/utils/standalone.js

```diff
@@ -31,7 +31,7 @@ const arch = (() => {
 
 module.exports = {
   resolveLatestTag: async () => {
-    const { body } = await got(
+    const { body } = await fetch(
```

Comment:
If `body` is a readable stream, then I'm not sure that the code below works: we're passing `body` to `JSON.parse`. I think the Response from `node-fetch` is a stream here, but therefore should we be calling the `json()` method on `body` instead of directly calling `JSON.parse`?

Reply:
I think here the body can not be the readable stream by seeing this.
``` javascript 
const tagName = await standaloneUtils.resolveLatestTag();
const latestVersion = tagName.slice(1);
```